lib/controller.rb

```diff
@@ -4,8 +4,16 @@
 require './lib/helpers/data_mapper'
 require 'pry'
 
-class SlowFood < Sinatra::Base
+class WeekThreeAssessment < Sinatra::Base
 
-  # Your code will go here.
+get '/' do
+@text = 'Kristoffer'
+erb :index
+  end
 
+  get '/name/:name' do
+  User.create(name:params[:name])
+
+
+  end
 end
```

Comment:
The indentation in this file needs to be cleaned up. We indent one tab (two spaces) for the "block" of a method. So:
```
get '/' do
  @text = 'Kristoffer'
  erb :index
end
```